src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -204,6 +204,10 @@ public void syncPostBatch(final String topicId, final List<BatchItem> batch) thr
         }
     }
 
+    private long createSendTimeout() {
```

Comment:
Well... I don't know. The purpose of hiding this is unclear for me. I'm fine with it, but it's strange